buddyboss/buddyboss-platform
# Mention is not working properly

### All Submissions:






### Changes proposed in this Pull Request:

This PR will fix mentions when no results found and curser goes to first position in the editor.

Fixes #735  .

### How to test the changes in this Pull Request:

1. Go to activity post form
2. try mentioning someone who is not in the site anywhere
3. curser should be stuck at same position

### Other information:
